staging/src/k8s.io/client-go/tools/record/event.go

```diff
@@ -135,6 +135,7 @@ func recordToSink(sink EventSink, event *v1.Event, eventCorrelator *EventCorrela
 		utilruntime.HandleError(err)
 	}
 	if result.Skip {
+		glog.Warningf("Dropping event '%#v' (rate limit exceeded!)", event)
```

Comment:
does it have to be rate limit?

Reply:
currently an event will be marked Skip only when it's be rate-limited.